### DISCUSSION TITLE
Пневма или же забытая тема с лоурайдерами.

### DISCUSSION THREAD

**Discussion by NOTGELB** [octothorp-team/public-dobrograd#657] [Предложения]
**Давным-давно в далекой далекой галактике...**
А точнее на старом форуме предлагали добавить пневму или же систему лоурайдинга. Тема достаточно быстро обрела популярность среди пользователей, и даже Ваня (Waniczka) что-то сделал на скорую руку дабы показать как это примерно должно выглядеть.
Время шло, в теме скапливалось все больше и больше ответов пользователей, но к сожалению о ней как-то все забыли и я решил поднять вновь данную мысль.  
**Она уже существует!**
Данную функцию уже реализовали на просторах воркшопа, и хочу сказать что она выглядит не дурно:
![image](https://user-images.githubusercontent.com/92695401/138167841-2d0b8939-c3ee-4e85-859a-15db41b6d379.png)
![image](https://user-images.githubusercontent.com/92695401/138167864-724d8984-c28f-4c74-a02e-176d86e95d60.png)
![image](https://user-images.githubusercontent.com/92695401/138167883-f16e5b62-8f39-416f-a08a-4e6b12ef14f5.png)
_К сожалению гифки не работают на гитхабе_

И смотря на это, назревает мысль, почему бы не использовать данный аддон:
https://steamcommunity.com/sharedfiles/filedetails/?id=2098560368

Что думаете? Надо или не надо? Хотели вы бы видеть подобное или нет?
А так, удачи.

**Comment by Ggman77** [comment]
Круто, но бесполезно
